components/filter/filter-tags.js

```diff
@@ -34,7 +34,6 @@ class FilterTags extends RtlMixin(LocalizeCoreElement(LitElement)) {
 		return [bodyCompactStyles, css`
 			:host {
 				display: inline-block;
-				width: 100%;
```

Comment:
This required that any consumer have this take up 100% of width, which was causing the issue in Nova where it could not be right aligned.